.github/workflows/maven-ci.yml

```
@@ -87,6 +90,9 @@ jobs:
     - name: OSGi-TCK Framework
       if: steps.changes.outputs.framework == 'true'
       run: mvn -B -V -Dstyle.color=always --file framework.tck/pom.xml clean verify
+    - name: Felix Gogo Shell
+      if: steps.changes.outputs.gogo == 'true'
+      run: mvn -B -V -Dstyle.color=always --file gogo/pom.xml clean install verify
```

```suggestion
      run: mvn -B -V -Dstyle.color=always --file gogo/pom.xml clean verify
```

`verify` includes `install`...

---

Thx for finding. Will update

---

It is the opposite, install includes verify. See https://maven.apache.org/guides/introduction/introduction-to-the-lifecycle.html
I just noticed this in email ;-)

---

You are wright. But. Why should i install

---

Yes sorry for confusion, I think verify is enough (as mentioned in the suggestion) as all other tasks are also use verify to execute the tests.